cmd/compgen/cmd/common/common.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"text/template"
+)
+
+type TemplateData struct {
+	Name        string
+	PackageName string
+}
+
+// Executes `go mod tidy`
```

Comment:
I can infer this by reading the code. Use comments to the tell the way "if it's needed."